components/TalkCard/TalkCard.js

```diff
@@ -0,0 +1,41 @@
+//import { data } from "../utils/data";
+import styled from "styled-components";
+import { data } from "../utils/data";
+
+export default function TalkCard() {
+  return (
+    <>
+      <h2>JURASSIC TALK 2022 Schedule</h2>
+      <StyledUl>
+        {data?.map((talk) => (
+          <StyledLi key={talk.id}>
+            <h4>{talk.title}</h4>
+            <h4>{talk.authors[0]}</h4>
+            <StyledDiv>
+              <p>{talk.session}</p>
+              <p>{talk.time}</p>
+              <p>{talk.room}</p>
+            </StyledDiv>
```

Comment:
I think `StyledDiv`it is not a good name. Maybe you find some more meaningful name for the styled component. Like `TalkInfoWrapper`

Reply:
Thank you for the helpful review!